doc/VitessOverview.md

```diff
@@ -44,7 +51,7 @@ Vitess improves a vanilla MySQL implementation in several ways:
   </tr>
   <tr>
     <td>Sharding is a process of partitioning your data to improve scalability and performance. MySQL lacks native sharding support, requiring you to write sharding code and embed sharding logic in your application.</td>
-    <td>Vitess uses range-based sharding. It supports both horizontal and vertical resharding, completing most data transitions with just a few seconds of read-only downtime. Vitess can even accommodate a custom sharding scheme that you already have in place.</td>
+    <td>Vitess supports a variety of sharding schemes. It can also migrate tables into different databases. It performs these functions non-intrusively, completing most data transitions with just a few seconds of read-only downtime. Vitess can even accommodate a custom sharding scheme that you already have in place.</td>
```

Comment:
I would readd resharding by mentioning it as scaling up.

Also, change the second sentence to avoid starting it with "It ..." as well.

Suggestion:

Vitess supports a variety of sharding schemes.
It can also migrate tables into different databases and scale up or down the number of shards.
These functions are performed non-intrusively, completing most data transitions with just a few seconds of read-only downtime.